dask/dataframe/utils.py

```diff
@@ -535,7 +535,7 @@ def _maybe_sort(a, check_index: bool):
 def _maybe_convert_string(a, b):
     import dask
 
-    if dask.config.get("dataframe.convert_string"):
+    if bool(dask.config.get("dataframe.convert-string")):
```

Comment:
IIUC this should already be a bool, so I don't think the `bool` call here is necessary. It's also not incorrect, just wanted to point this out